ios/FluentUI/Notification/FluentNotification.swift

```diff
@@ -63,9 +66,11 @@ public struct FluentNotification: View, ConfigurableTokenizedControl {
     ///   - attributedTitle: Optional attributed text to draw above the message area. If set, it will override the title parameter.
     ///   - image: Optional icon to draw at the leading edge of the control.
     ///   - trailingImage: Optional icon to show in the action button if no button title is provided.
+    ///   - trailingImageAccessibilityLabel: Optional localized accessibility label for the trailing image.
     ///   - actionButtonTitle:Title to display in the action button on the trailing edge of the control.
     ///   - actionButtonAction: Action to be dispatched by the action button on the trailing edge of the control.
-    ///   - messageButtonAction: Action to be dispatched by tapping on the toast/bar notification.
+    ///   - hasDefaultDismissButton: Bool to control if the Notification has a dismiss action by default.
+    ///   - messageButtonAction: Action to be dispatched by tapping on the toast/bar notification.   
     public init(style: MSFNotificationStyle,
```

Comment:
are we creating new convenence method to not disturb exsiting objc clients?

Reply:
No, this is just the SwiftUI one, it doesn't have @objc

Reply:
ah i see. but then i am confused why we adding this? can't swift client just override state themselves after creating a new instance?

Reply:
I think we need to add it here in order for the .presentNotification modifier to really be able to customize the state. So I think it's mostly for SwiftUI usage. Objective C is also able to just modify the state after init. So if the concern were just Objective C exposure of the state, we could drastically simplify a lot of our inits.

Reply:
Unlike other SwiftUI controls, `Notification` passes all of its state in `init` instead of via modifiers. I don't know why that is; these should probably all be broken out into separate functions in `NotificationModifiers`.

Reply:
I feel like that should be done in a separate PR though.